packages/medusa/src/api/routes/admin/reservations/index.ts

```diff
@@ -0,0 +1,77 @@
+import { Router } from "express"
+import { Note, ReservationItemDTO } from "../../../.."
+import { DeleteResponse, PaginatedResponse } from "../../../../types/common"
+import middlewares, { transformBody } from "../../../middlewares"
+import "reflect-metadata"
+import { AdminPostReservationsReq } from "./create-reservation"
+import { AdminPostReservationsReservationReq } from "./update-reservation"
+import { checkRegisteredModules } from "../../../middlewares/check-registered-modules"
+
+const route = Router()
+
+export default (app) => {
+  app.use(
+    "/reservations",
+    checkRegisteredModules({
+      inventoryService:
+        "Inventory is not enabled. Please add an Inventory module to enable this functionality.",
+    }),
+    route
+  )
+
+  route.get("/:id", middlewares.wrap(require("./get-reservation").default))
```

Comment:
todo: add list reservations too. 

We will need this for example to display all reservations currently held on a specific inventory item.